pytest_fixtures/component/satellite_auth.py

```diff
@@ -451,3 +452,32 @@ def enroll_ad_and_configure_external_auth(request, ad_data, target_sat):
     target_sat.execute(f'echo {settings.ldap.password} | {token_command}')
     target_sat.execute('chown root.apache /etc/httpd/conf/http.keytab')
     target_sat.execute('chmod 640 /etc/httpd/conf/http.keytab')
+
+    # enable the foreman-ipa-authentication feature
+    target_sat.execute('satellite-installer --foreman-ipa-authentication=true', timeout='20m')
```

Comment:
Would encourage to use InstallerCommand class here and verify if its enabled 
```suggestion
    result = target_sat.install(
        InstallerCommand('foreman-ipa-authentication true')
    )
    assert result.status == 0
```

Reply:
Updated.